test/unit/calculators/minimum_wage_calculator_test.rb

```diff
@@ -113,8 +113,17 @@ class MinimumWageCalculatorTest < ActiveSupport::TestCase
         end
       end
 
-      should "return false if age is lower than 24 or nil" do
+      should "return true if the age is 23 or over, and the date is on or after 2021-04-01" do
```

Comment:
Why do we have to support people checking that they're eligible for NLW on dates in the past?

Reply:
Ah, because you can check last years eligibility can't you. https://www.gov.uk/am-i-getting-minimum-wage/y GOTTIT!